### DISCUSSION TITLE
quasar components

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#9232] [General - Components / Directives / etc] [answered]
Hello all ,

can someone please help i wanted to know does quasar have a component that allow me to build something like this 

![component](https://user-images.githubusercontent.com/61178099/117810474-7ee5c200-b270-11eb-9f9a-9063f8077e9f.png)
)

Thanks

**Comment by hawkeye64** [comment, marked as answer]
@AkashRamtohul768  I think what you're looking for is a Floating Action Button: https://quasar.dev/vue-components/floating-action-button

**Comment by ghost** [comment]
Thanks Jeff i think you got it right



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
